src/Client/Core/DependencyInjection/DurableTaskClientBuilderExtensions.cs

```diff
@@ -101,4 +101,6 @@ public static IDurableTaskClientBuilder UseDefaultVersion(this IDurableTaskClien
         builder.Configure(options => options.DefaultVersion = version);
         return builder;
     }
+
+    // Large payload enablement moved to Microsoft.DurableTask.Extensions.AzureBlobPayloads package.
```

Comment:
Please remove this comment. I'm guessing it was created by a coding agent, and I noticed that they tend to leave these kinds of comments in the code, which is not good.

Reply:
updated